### DISCUSSION TITLE
Why not have a DirectML version for AMD GPU Windows builds?

### DISCUSSION THREAD

**Discussion by ArchAngelAries** [oobabooga/text-generation-webui#2030] [General]
Given that there's a DirectML version of A1111 Stable Diffusion that works great, one would think that it would be possible to adapt this to DirectML for AMD GPU Windows builds as well. I'm not skilled in coding, but it in my opinion it is a valid question/suggestion.

**Comment by BarfingLemurs** [comment]
https://github.com/oobabooga/text-generation-webui/issues/62#issuecomment-1536276353
dunno which llm used.
rocm + linux driver install on wsl2 should be faster, https://github.com/oobabooga/text-generation-webui/pull/1631

**Reply by nhk75** [reply]
Given that Stable Diffusion DirectML seems to work with AMD GPU fine now (on native windows 11 environment), I assume that everything is possible :)
